client/src/components/GeoChart/index.js

```diff
@@ -95,7 +95,7 @@ const GeoChart = ({ onClose, countryYearIndex, setCountryYearIndex, currentSearc
           <p id="rangeYearText">{countryYearIndexToYearMap[countryYearIndex]}</p> 
         </div> 
         <Chart
-          width={'2000px'}
+          width={'60vw'}
           height={'1500px'}
```

Comment:
This should probably be set to 'auto' for best behavior on multiple screens

Reply:
sure, just changed height to auto in latest commit